common/src/Polyhedron.h

```diff
@@ -392,12 +414,12 @@ class Polyhedron {
     bool checkEdges() const;
     bool checkEdgeLengths(const T minLength = vm::constants<T>::pointStatusEpsilon()) const;
     bool checkLeavingEdges(const Vertex* v) const;
-    
+
     void updateBounds();
 public: // Vertex correction and edge healing
     void correctVertexPositions(const size_t decimals = 0, const T epsilon = vm::constants<T>::correctEpsilon());
-    bool healEdges(const T minLength = vm::constants<T>::pointStatusEpsilon());
-    bool healEdges(Callback& callback, const T minLength = vm::constants<T>::pointStatusEpsilon());
+    bool healEdges(const T minLength = T(0.01));
```

Comment:
Maybe put this in the Polyhedron class, like:
`static const T DefaultMinEdgeLength = T(0.01);` ?

Good to get it out of `vm::constants<T>` because it's not a constant that's general to vector math, but it's specific to this polyhedron / convex hull code and Quake's scale.

Reply:
done